add-chain/e2e_test.go

```diff
@@ -121,15 +127,15 @@ func compareJsonFiles(t *testing.T, dirPath, testName, chainName string) {
 	require.Equal(t, diff, "", "expected json (-) does not match test json (+): %s", diff)
 }
 
-func checkConfigYaml(t *testing.T, testName, chainName string) {
+func checkConfigYaml(t *testing.T, testName, chainShortName string) {
 	expectedBytes, err := os.ReadFile("./testdata/superchain/configs/sepolia/expected_" + testName + ".yaml")
 	require.NoError(t, err, "failed to read expected.yaml config file: %w", err)
 
 	var expectedYaml map[string]interface{}
 	err = yaml.Unmarshal(expectedBytes, &expectedYaml)
 	require.NoError(t, err, "failed to unmarshal expected.yaml config file: %w", err)
 
-	testBytes, err := os.ReadFile(ymlConfigDir + chainName + ".yaml")
+	testBytes, err := os.ReadFile(ymlConfigDir + chainShortName + ".yaml")
```

Comment:
Since this PR changes the filenames written, the filenames in `cleanupTestFiles` should also use the `chainShortName` instead of `chainName`

Reply:
Good catch, [086a3d0](https://github.com/ethereum-optimism/superchain-registry/pull/337/commits/086a3d07a22bdb18a682c9f8e4cbe598ec3e796b)